packages/ui-react/src/containers/UpdatePaymentMethod/UpdatePaymentMethod.tsx

```diff
@@ -48,10 +48,10 @@ const UpdatePaymentMethod = ({ onCloseButtonClick }: Props) => {
     try {
       setPaymentError(undefined);
 
-      const successUrl = addQueryParams(window.location.href, { u: 'payment-method-success' });
-      const waitingUrl = addQueryParams(window.location.href, { u: 'waiting-for-payment' });
-      const cancelUrl = addQueryParams(window.location.href, { u: 'paypal-cancelled' });
-      const errorUrl = addQueryParams(window.location.href, { u: 'paypal-error' });
+      const successUrl = createURL(window.location.href, { u: 'payment-method-success' });
+      const waitingUrl = createURL(window.location.href, { u: 'waiting-for-payment' });
+      const cancelUrl = createURL(window.location.href, { u: 'paypal-cancelled' });
+      const errorUrl = createURL(window.location.href, { u: 'paypal-error' });
```

Comment:
No modalUrl call here. Why? :)

Reply:
I think i already know... `const location = useLocation(); `has not been defined here...

Reply:
Yes, and the urls should include the hostname, to be used from external (payment provider) sources